openwisp_users/apps.py

```diff
@@ -71,3 +82,28 @@ def update_organizations_dict(cls, instance, **kwargs):
         cache.delete(cache_key)
         # forces caching
         user.organizations_dict
+
+    def create_organization_owner(cls, instance, **kwargs):
+        if not kwargs.get('created') or not instance.is_admin:
+            return
+        owner_model = load_model('openwisp_users', 'OrganizationOwner')
+        model = kwargs.get('sender')
```

Comment:
why don't you call these two variables simply `OrganizationOwner` and `OrganizationUser`? Wouldn't this make the code more self explanatory?